azerothcore/azerothcore-wotlk
# fix(DB/creature_template): Enable swimming for Tooga

## Changes Proposed:
- Adds water to the InhabitType for [Tooga](https://tbc.wowhead.com/npc=5955/tooga), allowing him to swim during his [escort quest](https://tbc.wowhead.com/quest=1560/toogas-quest) if needed. This brings him into line with the Wowhead source saying he could swim, and improves on his behaviour before, where he would walk along the sea bottom and sometimes clip through the ground, failing the quest.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8115
- Closes https://github.com/chromiecraft/chromiecraft/issues/1832

## SOURCE:


[2007/12/08 (Patch 2.3.0)](https://www.wowhead.com/quest=1560/toogas-quest#comments:id=198090)
> For any druids doing this: Get to the water and then swim him up to steamwheedle while in aquatic form. He can swim at the same speed as you and most of the things you see in the water aren't hostile.

## Tests Performed:

- Ran SQL, no errors/warnings, 1 row changed as expected.
- Ran the quest again and observed him swimming now:

![WoWScrnShot_092721_212444](https://user-images.githubusercontent.com/81782124/134908887-22322a5f-5482-4c92-b079-0fe42f4e2cb8.jpg)
![WoWScrnShot_092721_212433](https://user-images.githubusercontent.com/81782124/134908897-fb105d62-e875-49d5-ab6a-92e55211ccf3.jpg)
![WoWScrnShot_092721_212805](https://user-images.githubusercontent.com/81782124/134908990-89abf288-fa8c-497e-b58d-ebb94044f857.jpg)

## How to Test the Changes:


1. Go to Tanaris and find Tooga - .go c 22831
2. Accept his quest
2. Head over to the sea by going mostly east.
3. Once in there, swim about, verifying he's not walking on the seabed, and head north to Steamwheedle.

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
